streams/src/test/java/org/apache/kafka/streams/state/KeyValueStoreTestDriver.java

```diff
@@ -201,7 +201,7 @@ private KeyValueStoreTestDriver(final StateSerdes<K, V> serdes) {
             logContext,
             new TaskId(0, 0),
             consumer,
-            new StreamsProducer(logContext, producer),
+            new StreamsProducer(producer, null != null, logContext, null),
```

Comment:
`null != null`?

Reply:
lol

Reply:
Oops! Auto-refactoring. I already made two passes to clean these up, looks like I missed one.